cves/kernel/CVE-2013-3223.yml

```diff
@@ -56,6 +66,11 @@ description_instructions: |
   Your target audience is people just like you before you took any course in
   security
 description:
+  When using recvfrom() or recvmsg() to receive data from an AX.25 socket, the
+  socker driver did not properly zero-initialize the source address struct
+  before copying it to userspace, leaking uninitialized padding bytes containing
+  kernel stack data.
```

Comment:
I would suggest to add in here a specific security issue like information disclosure here for more clarity.

Reply:
"leaking uninitialized padding bytes containing kernel stack data" should mean information disclosure.

Reply:
The description is meant to be succinct so I don't think explaining what an AX.25 socket is needed.